.github/workflows/integration-tests.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         brew install podman
         podman machine init
         podman machine start
-        echo "DOCKER_HOST=unix:///Users/runner/.local/share/containers/podman/machine/podman-machine-default/podman.sock" >> $GITHUB_ENV
+        echo "DOCKER_HOST=unix:///Users/runner/.local/share/containers/podman/machine/qemu/podman.sock" >> $GITHUB_ENV
```

Comment:
I think you could use [the following](https://github.com/containers/podman/blob/28573fa9d2a511eb3235533ddc8e97f7d8aa360a/pkg/machine/ignition.go#LL464C49-L464C93):
```
export DOCKER_HOST="unix://$(podman info -f "{{.Host.RemoteSocket.Path}}")"
```

(untested)

Reply:
will test out in a followup. just want to unblock some things for now.